couchbase/tests/common.py

```diff
@@ -195,32 +195,8 @@
 }
 
 BY_BUCKET_METRICS = [
-    'couchbase.by_bucket.ep_oom_errors',
-    'couchbase.by_bucket.ep_overhead',
-    'couchbase.by_bucket.ep_queue_size',
-    'couchbase.by_bucket.ep_vb_total',
-    'couchbase.by_bucket.mem_used',
-    'couchbase.by_bucket.vb_active_itm_memory',
-    'couchbase.by_bucket.vb_active_meta_data_memory',
-    'couchbase.by_bucket.vb_active_num',
-    'couchbase.by_bucket.vb_active_num_non_resident',
-    'couchbase.by_bucket.vb_active_queue_age',
-    'couchbase.by_bucket.vb_active_queue_size',
-    'couchbase.by_bucket.vb_pending_curr_items',
-    'couchbase.by_bucket.vb_pending_itm_memory',
-    'couchbase.by_bucket.vb_pending_meta_data_memory',
-    'couchbase.by_bucket.vb_pending_num',
-    'couchbase.by_bucket.vb_pending_num_non_resident',
-    'couchbase.by_bucket.vb_pending_queue_age',
-    'couchbase.by_bucket.vb_pending_queue_size',
-    'couchbase.by_bucket.vb_replica_curr_items',
-    'couchbase.by_bucket.vb_replica_itm_memory',
-    'couchbase.by_bucket.vb_replica_meta_data_memory',
-    'couchbase.by_bucket.vb_replica_num',
-    'couchbase.by_bucket.vb_replica_num_non_resident',
-    'couchbase.by_bucket.vb_replica_queue_age',
-    'couchbase.by_bucket.vb_replica_queue_size',
-    'couchbase.by_bucket.vb_total_queue_age',
+    # TODO I did not find a reliable way to make sure those metrics are always present.
```

Comment:
Should we just get rid of this list if there's nothing in it?

Reply:
sounds good to me, done!